cpu/cpupower.py

```diff
@@ -31,9 +33,9 @@ class cpupower(Test):
     :avocado: tags=cpu,power,privileged
     """
 
+    @skipIf(not IS_POWER_NV, "This test is not supported on  PowerVM platform")
```

Comment:
extra space here: `supported on  PowerVM`

Reply:
yeah i will remove that